### PR TITLE
Make sure admin audit logs are always written

### PR DESCRIPTION
**Pull request by rullzer**
Else if you loglevel is 3 for example you never see anything in the
logs.

@wiswedel @schiessle as I think you both ran into this as well. Quick test is appreciated.

### CODE REVIEW

File: apps/admin_audit/lib/AppInfo/Application.php [hunk 1]
```diff
@@ -76,6 +79,9 @@ public function initLogger() {
 			$this->logger = $c->getLogger();
 			return;
 		}
+
+		$$this->logLevel = $config->getSystemValueInt('loglevel', 2);
```
Comment (blizzz) [inline apps/admin_audit/lib/AppInfo/Application.php]: Was my coffee too Irish or is there a dollar sign too many :)

Comment (rullzer) [reply apps/admin_audit/lib/AppInfo/Application.php]: why not both :wink:

**Comment by kesselb** [comment]
`'log.condition' => ['apps' => ['admin_audit']];` if you need a solution today. Also I don't like changing the log level like that. It's weird.

File: apps/admin_audit/lib/AppInfo/Application.php [hunk 2]
```diff
@@ -61,6 +61,9 @@ class Application extends App {
 	/** @var ILogger */
 	protected $logger;
 
+	/** @var int */
+	private $logLevel;
```
Comment (wiswedel) [inline apps/admin_audit/lib/AppInfo/Application.php]:
```suggestion
	private $loglevel;
```

File: apps/admin_audit/lib/AppInfo/Application.php [hunk 3]
```diff
@@ -76,6 +79,9 @@ public function initLogger() {
 			$this->logger = $c->getLogger();
 			return;
 		}
+
+		$this->logLevel = $config->getSystemValueInt('loglevel', 2);
```
Comment (wiswedel) [inline apps/admin_audit/lib/AppInfo/Application.php]:
```suggestion
		$this->loglevel = $config->getSystemValueInt('loglevel', 2);
```

File: apps/admin_audit/lib/AppInfo/Application.php [hunk 4]
```diff
@@ -105,7 +111,7 @@ protected function registerHooks() {
 	}
 
 	protected function userManagementHooks() {
-		$userActions = new UserManagement($this->logger);
+		$userActions = new UserManagement($this->logger, $this->logLevel);
```
Comment (wiswedel) [inline apps/admin_audit/lib/AppInfo/Application.php]:
```suggestion
		$userActions = new UserManagement($this->logger, $this->loglevel);
```

**Comment by rullzer** [comment]
> `'log.condition' => ['apps' => ['admin_audit']];` if you need a solution today. Also I don't like changing the log level like that. It's weird.

right that might be cleaner for now. Yeah I agree that maybe abusing the basic logger for this is not ideal.

**Pull request closed without merging**